src/main/java/com/limechain/config/SystemInfo.java

```diff
@@ -27,7 +27,7 @@ public class SystemInfo {
     private String hostVersion;
     private final BigInteger highestBlock;
 
-    public SystemInfo(HostConfig hostConfig, Network network, SyncState syncState) {
+    public SystemInfo(HostConfig hostConfig, NetworkService network, SyncState syncState) {
```

Comment:
Rename to `networkService`

Reply:
actually you can leave it as is, didn't see that Network class is getting removed